lib/resources/Customers.js

```diff
@@ -144,6 +144,12 @@ module.exports = StripeResource.extend({
     urlParams: ['customerId', 'sourceId'],
   }),
 
+  validateSource: stripeMethod({
+    method: 'POST',
+    path: '/{customerId}/sources/{sourceId}/validate',
```

Comment:
@wbronitsky Cool! And I think the discrepancy might have made it into the function naming as well. Maybe consider a rename from `validateSource` to `verifySource`.